app/src/main/AndroidManifest.xml

```diff
@@ -46,7 +46,7 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-            <intent-filter>
+            <intent-filter android:autoVerify="true">
```

Comment:
Can you mention in the readme to add the SHA256 on Firebase please?

Reply:
It's actually not needed, I removed my SHA256 and it still worked.

Reply:
Perfect :)

Reply:
But I will remove the autoVerify, it is not needed either I think the way we do things now.